example/index.html

```diff
@@ -76,53 +63,42 @@
 
         // Now we setup our event handlers, to respond to the authentication
         // state that Stormpath provides.
-
         stormpath.on('authenticated', function () {
-
           // This event is fired once we know that the user is authenticated,
           // this can fire on login, or on page reload when the user is already
           // logged in.
-
           console.log('We\'re authenticated!');
 
           // Hide the Stormpath widget
-
           stormpath.remove();
```

Comment:
This is actually not necessary, unless we demo the widget in DOM mode (not modal).